app/assets/javascripts/discourse/app/components/quote-button.js

```diff
@@ -1,3 +1,6 @@
+import { propertyEqual } from "discourse/lib/computed";
```

Comment:
I think the name of this file is no longer representative of what is going on here.

Reply:
yes I thought about it, but not going to rename this in this PR, there's already share which is unrelated too